_config.yml

```diff
@@ -34,7 +34,11 @@ exclude:
   - tests
   - assets/vendor/selectivizr/tests
   - assets/vendor/zeroclipboard/test
-  
+
 gems:
   - jekyll-sitemap
   - jekyll-redirect-from
+
+sass:
+    sass_dir: _sass
+    style: :compressed
```

Comment:
Might as well have `script/server` refer to a dev config file that reads:

``` yaml
sass:
  http_path: ''
  style: :expanded
```

Keep the SASS expanded in dev, compressed in prod.